### PR TITLE
Hot fix - Convection report vars initialization

### PR DESCRIPTION
**Pull request by mitchute**
Pull request overview
---------------------
 - Fixes another issue with the convection report vars not being initialized.

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [ ] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [ ] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [ ] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [ ] If changes fix a defect, the fix should be demonstrated in plots and descriptions
 - [ ] If any defect files are updated to a more recent version, upload new versions here or on DevSupport
 - [ ] If IDD requires transition, transition source, rules, ExpandObjects, and IDFs must be updated, and add IDDChange label
 - [ ] If structural output changes, add to output rules file and add OutputChange label
 - [ ] If adding/removing any LaTeX docs or figures, update that document's CMakeLists file dependencies

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/ConvectionCoefficients.cc [hunk 1]
```diff
@@ -5729,6 +5723,9 @@ void MapExtConvClassificationToHcModels(EnergyPlusData &state, int const SurfNum
                         format("MapExtConvClassificationToHcModels: caught unknown outdoor surface classification: {}",
                                state.dataSurface->SurfOutConvClassification(SurfNum)));
     }
+
+    // Set report var after surface has been classified
+    state.dataSurface->SurfOutConvClassificationRpt(SurfNum) = static_cast<int>(state.dataSurface->SurfOutConvClassification(SurfNum));
```
Comment (rraustad) [inline src/EnergyPlus/ConvectionCoefficients.cc:5728]: Better here where any rework will include this.

File: src/EnergyPlus/SurfaceGeometry.cc [hunk 2]
```diff
@@ -1024,9 +1024,11 @@ namespace SurfaceGeometry {
             state.dataSurface->SurfExtConvCoeffIndex(SurfNum) = 0;
             state.dataSurface->SurfTAirRef(SurfNum) = 0;
             state.dataSurface->SurfIntConvClassification(SurfNum) = ConvectionConstants::InConvClass::Invalid;
+            state.dataSurface->SurfIntConvClassificationRpt(SurfNum) = static_cast<int>(ConvectionConstants::InConvClass::Invalid);
             state.dataSurface->SurfIntConvHcModelEq(SurfNum) = 0;
             state.dataSurface->SurfIntConvHcUserCurveIndex(SurfNum) = 0;
             state.dataSurface->SurfOutConvClassification(SurfNum) = ConvectionConstants::OutConvClass::Invalid;
+            state.dataSurface->SurfOutConvClassificationRpt(SurfNum) = static_cast<int>(ConvectionConstants::OutConvClass::Invalid);
```
Comment (rraustad) [inline src/EnergyPlus/SurfaceGeometry.cc:1031]: I am curious if this will cause (corrective) diffs now compared to yesterday.

**Review by rraustad — APPROVED** [review]
Might want to see a CI result for this one.

File: src/EnergyPlus/SurfaceGeometry.cc (hunk 2 above)
Comment (mitchute) [reply src/EnergyPlus/SurfaceGeometry.cc:1031]: Unknown. I just ran regressions locally (on Mac) and see zero difference between current develop and this branch. I don't have a way to run regressions on other platforms, so I too am curious if it helps.

**Comment by mitchute** [comment]
> Might want to see a CI result for this one.

I think I got too trigger-happy and turned decent off for the last commit. I had to make a quick clang-format push, so I'm not sure if it's going to pick up the one before that or not. I wonder if I can amend the commit message and push? @Myoldmopar do you think decent would pick that up?

**Comment by mjwitte** [comment]
@mitchute You probably need a new commit to get this to fire off CI. Just merge in develop again now and it should be good.

File: src/EnergyPlus/SurfaceGeometry.cc (hunk 2 above)
Comment (rraustad) [reply src/EnergyPlus/SurfaceGeometry.cc:1031]: Well, the right 2 example files had diffs but this doesn't make any sense. The new variables SurfIn/OutConvClassificationRpt aren't acted on by anything, they just store the values of the old variables and report them. I wonder if the functions where the old variables are calculated are called every time step? Anyway, if it fixes diffs then so be it.

**Comment by rraustad** [comment]
Note to check if these 2 diffs files show up in a near future PR.

**Comment by mitchute** [comment]
Thanks, @rraustad.

**Pull request merged**